debezium-core/src/main/java/io/debezium/converters/CloudEventsMaker.java

```diff
@@ -275,4 +281,52 @@ public String ceId() {
                     + ";event_serial_no:" + recordParser.getMetadata(RecordParser.SqlserverRecordParser.EVENT_SERIAL_NO_KEY);
         }
     }
+
+    /**
+     * CloudEvents maker for records produced by Oracle connector.
+     */
+    public static final class OracleCloudEventsMaker extends CloudEventsMaker {
```

Comment:
See above: let's move those classes to the respective connector projects and expose them via some service contract.